apps/sel4test-tests/src/tests/breakpoints.c

```diff
@@ -263,8 +264,11 @@ test_debug_set_data_breakpoint(struct env *env)
     }
     return sel4test_get_result();
 }
+/* This test is flaky under simulation. See also #43  */
```

Comment:
should better day "GitHub Issue #43" to give a bit more context that this numer refers to.